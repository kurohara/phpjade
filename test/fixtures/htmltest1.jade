html
  body
    - var t = 'abc';
    div #{t}
