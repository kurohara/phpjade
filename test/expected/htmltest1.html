<html><body><div>abc</div></body></html>